docs/DebuggingGuide.md

```diff
@@ -30,7 +30,13 @@ Additional setup guidance can be found in the [Contributing.md](https://github.c
     `git checkout -b informative-branch-name`
 
 ## Debugging
-0. Install the VS SARIF viewer for [VS 2019](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) or [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+0. Install the VS SARIF viewer for [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+   
+   Select `Debug` from the `Solution Configurations` menu:
+
+   ![Solution Configurations: Debug](DebugBuild.png)
+   
+   Then run `Build Solution` from the `Build` menu.
```

Comment:
@HulonJenkins, let me know if the image is helpful here. Also, if you want to check formatting for a documentation update (ie. when someone adds an image), it can be helpful to enlist in their branch and open in VS Code. Alternatively, you can ask for someone to add a screenshot of the result in the description.

Reply:
It helped me realize I didn't have the solution open once I opened the folder in VS. I made a separate comment about this issue though.